ms365_accountcreator/translations/de/LC_MESSAGES/messages.po

```diff
@@ -101,13 +111,18 @@ msgid ""
 "I consent that my st****** email is saved to prevent duplicate Account "
 "creation."
 msgstr ""
-"Ich stimme zu, dass meine st****** E-Mail Adresse gespeichert wird um zu "
-"verhindern, das mehrere Accounts für einen Student angelegt werden "
-"können."
+"Ich stimme zu, dass meine eingegebene st****** E-Mail Adresse verarbeitet"
+" und gespeichert wird um das generierte Passwort zu verschicken und um "
+"sicherzustellen, dass maximal ein Account pro Studi angelegt wird. Mir "
```

Comment:
```suggestion
"sicherzustellen, dass maximal ein Account pro E-Mail Adresse angelegt wird. Mir "
```

Ansonsten genderneutral Student*in anstatt von Studi

Reply:
Das macht halt inhaltlich wengier Sinnd mit E-Mail-Addresse. Studi wird häufiger als genderneutrale singular Form von Studierende verwendet.

Reply:
Aber Studi wirkt halt echt komisch in so einem Text. In so Texten sollte man auch immer so deutlich wie möglich sein.

Reply:
```suggestion
"sicherzustellen, dass maximal ein Account pro Student*in angelegt wird. Mir "
```

Reply:
Ok, das war ja mein alternativvorschlag